src/plone/exportimport/utils/content/import_helpers.py

```diff
@@ -117,9 +117,17 @@ def get_obj_instance(item: dict, config: types.ImporterConfig) -> DexterityConte
         logger.debug(f"{config.logger_prefix} Will update {new}")
     else:
         factory_kwargs = item.get("factory_kwargs", {})
-        new = unrestricted_construct_instance(
-            item["@type"], container, item["id"], **factory_kwargs
-        )
+        repo_tool = api.portal.get_tool("portal_repository")
+        # Temporarily disable versioning, otherwise the first version
+        # is basically nothing, it does not even have a title.
+        orig_versionable = repo_tool.getVersionableContentTypes()
+        repo_tool.setVersionableContentTypes([])
+        try:
+            new = unrestricted_construct_instance(
+                item["@type"], container, item["id"], **factory_kwargs
+            )
+        finally:
+            repo_tool.setVersionableContentTypes(orig_versionable)
```

Comment:
This looks like it would end up marking portal_repository as _p_changed and including it in the transaction, even if the new object state is unchanged.

It might actually be cleaner to patch repo_tool.isVersionable using unittest.mock??

Reply:
You are right, the repo tool would be marked as changed.

It feels strange to use unittest code in production code.  But the patch method works fine and is clean.  Done.